src/popup/utils/constants.ts

```diff
@@ -447,3 +447,10 @@ export const INPUT_MESSAGE_STATUSES = {
   warning: 'warning',
   error: 'error',
 } as const;
+
+export const TX_FUNCTION_TIP = 'tip';
+export const TX_FUNCTION_RETIP = 'retip';
+export const TX_FUNCTION_TRANSFER = 'transfer';
+export const TX_FUNCTION_SENT = 'sent';
+export const TX_FUNCTION_RECEIVED = 'received';
+export const TX_FUNCTION_DEPOSIT = 'deposit';
```

Comment:
It's really nice that we have TypeScript right now, because some inconsistencies can be noticed. Like for example the possible values for the `transaction.tx.function`. Unitil now it should use the `TxFunction` type located in `src/types/index.ts`.
But here you listed other options, in which some of them are the same.
Please establish the correct values for the `transaction.tx.function`, update the `TxFunction` type and use it to create the list of constants. Or (which I suggest) do the opposite:

```
export const TX_FUNCTION = {
  tip: 'tip',
  retip: 'retip',
  ...
} as const;
```

and then in the types file:
```
import { TX_FUNCTION } ...

export type TxFunction = ObjectValues<typeof TX_FUNCTION>;
```